tests/countries/test_libya.py

```diff
@@ -0,0 +1,305 @@
+        self.assertNoHolidayName(name, range(1988, 2012))
+
+    def test_independence_day(self):
+        name = "عيد الاستقلال"
+        self.assertHolidayName(name, (f"{year}-12-24" for year in range(2012, 2050)))
+        self.assertNoHolidayName(name, range(1988, 2012))
+
+    def test_islamic_new_year(self):
+        name = "عيد رأس السنة الهجرية"
+        self.assertHolidayName(
+            name,
+            "2021-08-10",
+            "2022-07-30",
+            "2023-07-19",
+            "2024-07-07",
+            "2025-06-26",
+        )
+        self.assertHolidayName(name, self.no_estimated_holidays, range(2012, 2050))
+        self.assertHolidayName(name, self.workday_holidays, range(1988, 2012))
+        self.assertNoHolidayName(name, self.no_estimated_holidays, range(1988, 2012))
+
+    def test_prophets_birthday(self):
+        name = "ذكرى المولد النبوي الشريف"
+        self.assertHolidayName(
+            name,
+            "2020-10-29",
+            "2021-10-19",
+            "2022-10-08",
+            "2023-09-27",
+            "2024-09-15",
+        )
+        self.assertHolidayName(name, self.no_estimated_holidays, range(1988, 2050))
+
+    def test_eid_al_fitr(self):
+        name = "عيد الفطر"
+        self.assertHolidayName(
+            name,
+            "2023-04-21",
+            "2023-04-22",
+            "2023-04-23",
+            "2024-04-10",
+            "2024-04-11",
+            "2024-04-12",
+            "2025-03-31",
+            "2025-04-01",
+            "2025-04-02",
+        )
+        self.assertHolidayName(name, self.no_estimated_holidays, range(1988, 2050))
+
+    def test_day_of_arafah(self):
+        name = "يوم عرفة"
+        self.assertHolidayName(
+            name,
+            "2020-07-30",
+            "2021-07-19",
+            "2022-07-08",
+            "2023-06-27",
+            "2024-06-15",
+            "2025-06-05",
+        )
+        self.assertHolidayName(name, self.no_estimated_holidays, range(1988, 2050))
+
+    def test_eid_al_adha(self):
+        name = "عيد الأضحى"
+        self.assertHolidayName(
+            name,
+            "2023-06-28",
+            "2023-06-29",
+            "2023-06-30",
+            "2024-06-16",
+            "2024-06-17",
+            "2024-06-18",
+            "2025-06-06",
+            "2025-06-07",
+            "2025-06-08",
+        )
+        self.assertHolidayName(name, self.no_estimated_holidays, range(1988, 2050))
+
+    def test_syrian_revolution_day(self):
+        name = "عيد ثورة سوريا"
+        self.assertHolidayName(
+            name,
+            self.workday_holidays,
+            "1988-03-08",
+            "1989-03-08",
+            "2009-03-08",
+            "2010-03-08",
+            "2011-03-08",
+        )
+        self.assertHolidayName(name, self.workday_holidays, range(1988, 2012))
+        self.assertNoHolidayName(name, self.workday_holidays, range(2012, 2050))
+        self.assertNoHolidayName(name, range(1988, 2050))
```

Comment:
```suggestion
        self.assertNoHolidayName(name)
        self.assertHolidayName(name, self.workday_holidays, (f"{year}-03-08" for year in range(1988, 2012)))
        self.assertNoHolidayName(name, self.workday_holidays, range(2012, 2050))
```
And the same for rest of fixed date holidays.